### PR TITLE
Error 404 page for Client side.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
#290 solved. @aka-jain Please review the PR.

### CODE REVIEW

**Comment by aka-jain** [comment]
@RishabhJain2018 : Why you are using image for 404 message? Please write text (using html, css) instead of using image. Thanks.

**Comment by RishabhJain2018** [comment]
@aka-jain Sure . I will update that.

**Comment by RishabhJain2018** [comment]
@aka-jain here is the screenshot. Please review the PR.
error

File: frontend/src/views/web/error-pages/error-404.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          Error 404
+        </div>
+
+        <div class="message">
+          Woops. Looks like this page doesn't exist.
+        </div>
+      </div>
+
+      <div class="home">
+        Go Back to <a ui-sref="home"> www.evalai.cloudcv.org </a>
```
Comment (deshraj) [inline frontend/src/views/web/error-pages/error-404.html]: Can you change the link to `evalai.cloudcv.org`

File: frontend/src/views/web/error-pages/error-404.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  <link rel="stylesheet" type="text/css" href="/dist/css/main.css">
+
+</head>
+
+<body>
+  <div ui-view>
+
+    <div class="wrapper">
+      <div class="error_container">
+        <div class="error">
```
Comment (aka-jain) [inline frontend/src/views/web/error-pages/error-404.html]: @RishabhJain2018 : 'error' class name is common, so please change the name to some unique name like fof_error, fof_msg etc.

File: frontend/src/views/web/error-pages/error-404.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          Error 404
+        </div>
+
+        <div class="message">
```
Comment (aka-jain) [inline frontend/src/views/web/error-pages/error-404.html]: Change the class name please to fof_msg.

File: frontend/src/views/web/error-pages/error-404.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```
Comment (aka-jain) [inline frontend/src/views/web/error-pages/error-404.html]: @RishabhJain2018 : A big change you have to do here. Actually you have to make a module for 404 and call it using states. You don't need to write a whole html.
Briefly explain, every html is like a module which gets rendered in index.html, so every script and styles should only be declared in index.html. But you made a new html for 404 which is not a required solution. 
Please go through other html files like team or dashboard from which you can get an idea how to write a module.
Let me know if you need more help.

File: frontend/src/views/web/error-pages/error-404.html (hunk 1 above)
Comment (RishabhJain2018) [inline frontend/src/views/web/error-pages/error-404.html]: okk sure.

File: frontend/src/views/web/error-pages/error-404.html (hunk 2 above)
Comment (RishabhJain2018) [inline frontend/src/views/web/error-pages/error-404.html]: okk sure.

**Comment by RishabhJain2018** [comment]
@aka-jain This is the screenshot of the Error 404 page with the changes. Please review the PR.
error

File: frontend/src/views/web/error-pages/error-404.html (hunk 4 above)
Comment (RishabhJain2018) [inline frontend/src/views/web/error-pages/error-404.html]: okk sure.

**Comment by aka-jain** [comment]
@RishabhJain2018 : Can you also check it for the URL "127.0.0.1:8888/error-test-best". I know it will work but just curious to know the result :D

**Comment by RishabhJain2018** [comment]
@aka-jain The result of the url `127.0.0.1:8888/error-test-best` is the landing page.

**Comment by aka-jain** [comment]
@RishabhJain2018 : That's wrong. It should be 404 page. Can you check your routes again?

**Comment by RishabhJain2018** [comment]
@aka-jain okk sure.

**Comment by aka-jain** [comment]
@RishabhJain2018 : may be this can help you http://www.michaelware.net/post/2014/04/23/AngularJS-Routing-Using-Otherwise-without-redirectTo-(or-how-to-make-a-404-page).aspx

**Comment by RishabhJain2018** [comment]
@aka-jain Thank you for the resource. Please review the PR.

**Comment by aka-jain** [comment]
@RishabhJain2018 : Great work. Thanks :)
@deshraj : Merge it please!

**Pull request merged**